x/slashing/keeper/epoch.go

```diff
@@ -0,0 +1,123 @@
+package keeper
+
+// +TODO just copied from staking module
+// There should be a way to simplify this
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	db "github.com/tendermint/tm-db"
+)
+
+// keys
+var (
+	NextEpochActionID      = []byte("next_epoch_action_id")
+	EpochActionQueuePrefix = "epoch_action"
```

Comment:
Don't we need to save nextEpochIndex somewhere?

Reply:
Also, can we call it EpochNumber instead of EpochIndex

Reply:
How many EpochNumbers will exist? ( It should be constant like EpochNumbers = 2 which means there are two types of epochs?)

Reply:
I think a constant number makes sense, treating it as a proper queue. We should add helper methods for debuggability tho, to make it easy to get the actual epoch number.

Reply:
I'm still confused by this naming though.

EpochNumber to me means which epoch are we in, and this can scale infinitely.

If I understand the prior comment, whats implemented is in the range {0, 1}, e.g. an enum of `NEXT_EPOCH, CURRENT_EPOCH`